fastlane/lanes/build.rb

```diff
@@ -165,7 +171,7 @@
     archive_zip_path = File.join(PROJECT_ROOT_FOLDER, 'WordPress.xarchive.zip')
     zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
 
-    version = options[:beta_release] ? ios_get_build_version : get_app_version
+    version = options[:beta_release] ? current_build_code : current_app_version
```

Comment:
Should this be:

```suggestion
    version = options[:beta_release] ? current_build_code : current_release_version
```

?

I don't see any definition of `current_app_version`.

Reply:
Fixed https://github.com/wordpress-mobile/WordPress-iOS/commit/9db971b99eb4dcc172c69356baf96cdde7729c96